client/components/Layout.js

```diff
@@ -0,0 +1,20 @@
+import Head from "next/head";
+import Navbar from "./Navbar";
+import React from "react";
+
+const Layout = props => (
+  <div>
```

Comment:
FYI, you can use fragments instead of `div` in this case.

```jsx
<>
  <Head>
    ...
  </Head>
  <Navbar />
  <div>{props.children}</div>
</>
```

Reply:
ok cool, thanks @hitochan777.  Do you want me to merge this then?

Reply:
Either is ok. We can fix this in the future PRs.